PayPal/src/test/java/com/braintreepayments/api/PayPalVaultRequestUnitTest.java

```diff
@@ -80,17 +84,33 @@ public void parcelsCorrectly() {
 
         assertEquals("en-US", result.getLocaleCode());
         assertEquals("Billing Agreement Description",
-                result.getBillingAgreementDescription());
+            result.getBillingAgreementDescription());
         assertTrue(result.getShouldOfferCredit());
         assertTrue(result.isShippingAddressRequired());
         assertTrue(result.isShippingAddressEditable());
         assertEquals("Postal Address", result.getShippingAddressOverride()
-                .getRecipientName());
+            .getRecipientName());
         assertEquals(PayPalRequest.LANDING_PAGE_TYPE_LOGIN, result.getLandingPageType());
         assertEquals("Display Name", result.getDisplayName());
         assertEquals("123-correlation", result.getRiskCorrelationId());
         assertEquals("merchant_account_id", result.getMerchantAccountId());
         assertEquals(1, result.getLineItems().size());
         assertEquals("An Item", result.getLineItems().get(0).getName());
     }
+
+    @Test
+    public void createRequestBody_sets_userAuthenticationEmail_when_not_null() throws JSONException {
+        String payerEmail = "payer_email@example.com";
+        PayPalVaultRequest request = new PayPalVaultRequest();
+
+        request.setUserAuthenticationEmail(payerEmail);
+        String requestBody = request.createRequestBody(
+            mock(Configuration.class),
```

Comment:
🧶 : We do have some canned `Authorization` and `Configuration` strings in the `Fixtures` if we ever want to reduce mocks, but these work well here. Just a [callout](https://github.com/braintree/braintree_android/blob/a25c8122e427bc7e6a2f43b68b9cc10a5d6602e2/BraintreeCore/src/test/java/com/braintreepayments/api/ApiClientUnitTest.kt#L29) to a pattern that exists in the codebase.

Reply:
ohh nice! i don't think we were aware of the `Fixtures` when writing these tests. we can align this once the feature is out the door 👍

Reply:
Yeah definitely this is good now.